index.html

```diff
@@ -7410,6 +7408,16 @@ <h2 id="Performance">
           animation frame-rates do. Depending on the application, a reasonable
           latency can be from as low as 3-6 milliseconds to 25-50 milliseconds.
         </p>
+        <p>
+          Implementations will generally seek to minimize overall latency.
+        </p>
+        <p>
+          Along with minimizing overall latency, implementations will generally seek to minimize
+          the difference between an <a><code>AudioContext</code></a>'s <code>currentTime</code>
+          and a <a><code>AudioProcessingEvent</code></a>'s <code>playbackTime</code>.
```

Comment:
`and an <a><code>AudioProcessingEvent</code></a>'s ...`
instead of
`and a <a><code>AudioProcessingEvent</code></a>'s ...`
?